stable/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 0.2.3
+version: 0.3.0
```

Comment:
This is a breaking change so should bump this up to 1.x.x

Reply:
Updated to 1.0.0 :smile_cat: